pages/index.js

```diff
@@ -0,0 +1,171 @@
+import { useEffect } from "react";
+import Head from "next/head";
+import anime from "animejs";
+
+const Home = () => {
+  useEffect(() => {
+    anime({
+      targets: "div",
+      translateX: 250,
```

Comment:
```suggestion
```

Reply:
Turns out better if that's what you actually wanted?
![image](https://user-images.githubusercontent.com/13788498/78820288-a47c1880-7a2b-11ea-8fb2-4522e4793159.png)

Reply:
Oh yeah, it's a dumpster fire! 

All that I wanted to prove was that it animated. 

The final frame is a hot mess, but seeing as it's just a POC and going to be deleted, it didn't seem worth expending effort on it.

Reply:
So: if you see that animation, that's all this PR was about. Not the final layout.